_opensearch/rest-api/count.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title: Count
+parent: REST API reference
+nav_order: 150
+---
+
+# Count
+Introduced 1.0
+{: .label .label-purple }
+
+The count API gives you quick access to the number of documents that match a query.
+You can also use it to check the document count of an index, data stream, or cluster.
+
+
+## Example
+
+To see the number of documents that match a query:
+
+```json
+GET opensearch_dashboards_sample_data_logs/_count
+{
+  "query": {
+    "term": {
+      "response": "200"
+    }
+  }
+}
+```
+
+You can also use the search API for the same result:
```

Comment:
I think it's worth being explicit here that _count is really just search without the results. This is a great sample call, but can we introduce it with something more like, "The following call to the search API produces equivalent results"?